src/openqaoa-core/algorithms/workflow_properties.py

```diff
@@ -262,6 +261,54 @@ def __init__(
     #     self._cvar_alpha = value
 
 
+class ErrorMitigationProperties(WorkflowProperties):
+    """
+    TODO
+    """
+
+    def __init__(
+        self,
+        error_mitigation_technique: Optional[str] = None,
+        n_batches: Optional[int] = 10,
```

Comment:
Two comments here:
- How did we come up with the default value of 10 for `n_batches` 
- If the default value of `n_batches` is 10, you don't need to use `Optional` type hinting.

Reply:
I think that the number of batches should be of the same order of magnitude as the system size. But it's more or less arbitrary.

Reply:
Let's add a comment to justify this default value. In the docstring itself should work